### PR TITLE
Add feedback support link

### PR DESCRIPTION
**Pull request by nylon22**

### CODE REVIEW

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://603d208b5428f90e2687eb21--condescending-leavitt-66e607.netlify.app

File: packages/feedback/src/Feedback.js [hunk 1]
```diff
@@ -36,20 +39,26 @@ const Feedback = ({
...
           onFeedbackSent={onFeedbackSent}
           prompt={prompt}
-          isOpen={isOpen}
+          isOpen={feedbackIsOpen}
           zIndex={zIndex}
-          toggle={() => toggle()}
+          toggle={() => feedbackToggle()}
           name={appName}
           analytics={analytics}
+          showSupport={showSupport}
```
Comment (chrishavekost) [inline packages/feedback/src/Feedback.js:47]: I think we should also list the `supportZIndex` prop here to be clear even though `FeedbackModal` will destructure it from `{...formProps}`. The rest LGTM to me though!

**Review by chrishavekost — APPROVED** [review]

File: packages/feedback/src/Feedback.js (hunk 1 above)
Comment (nylon22) [reply packages/feedback/src/Feedback.js:47]: See 0450fa4e612688211cc7c61ce0b0c8591dbb42ce

**Comment by availity-bot-cicd** [comment]
🚀 Netlify deployed **condescending-leavitt-66e607** as draft

https://603d327ab2fe59268749b4c6--condescending-leavitt-66e607.netlify.app

**Review by cwhittemore1 — APPROVED** [review]

**Pull request merged**